Update Accessibility Details

titusfortner opened the pull request:
### Description
* Add image of what the integration looks like
* Add implementation details for Ruby, Python & Java with: 
  * Requirements
  * Usage
  * Example Code 
* Add links to all relevant code bases, projects, and documentation

### Motivation and Context
Per the conversation in this ticket: https://saucedev.atlassian.net/browse/TCD-1668

Focusing solely on wdio is useful to only ~10% of our user base.

The vast majority of the existing content for wdio is a rewrite of the information found at https://webdriver.io/docs/gettingstarted & https://webdriver.io/docs/sauce-service and I think it only complicates the information we are trying to demonstrate by including it in this document.

Also, there is a more complicated example in the wdio code for `getAxeResults()`, but there isn't any example or info on why to do that, or what can be done with it, so I removed it. Unlike the other languages, I don't have an actual code example to point to. Maybe @wswebcreation can give guidance. Finally, we link to https://www.npmjs.com/package/@axe-core/webdriverio but I don't understand how to match the documentation they have there with what we are providing here (different variables ; but maybe that's because I'm not a JS dev.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Resolves: [TCD-1668](https://saucedev.atlassian.net/browse/TCD-1668)

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -11,6 +11,7 @@ keywords:
 import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <p><small><span class="highlight beta">BETA</span></small></p>
```
```suggestion

```

wswebcreation reviewed (approved):
Hi @titusfortner and @spider-sauce 

Thanks for the PR and making this _better_ and more usable for our customers. I do have 1 question. Do we have more information about what the impact of a poor accessible website is, so we can explain in a few sentences how this can help improving that?
Maybe we have a video or something.

titusfortner commented:
Just added 3 things:

1. Changed the headers to just be bold text to keep the TOC clean (we could put them back and remove the TOC if that's better)
2. Made it more clear why someone should visit the Deque docs ( @wswebcreation's concern) 
3. SeleniumAxeDotnet just released a version that has been verified to work with our product, so I added more info about it.

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -11,44 +11,91 @@ keywords:
 
-Add the following dependencies to your `package.json` file:
+We highly recommend using the Java [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/).
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -11,44 +11,91 @@ keywords:
-"@wdio/sauce-service"
+__Requirements__
+
+Add this to your `pom.xml`
```
```suggestion
Add this to your `pom.xml`:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -11,44 +11,91 @@ keywords:
+
+__Usage__
+
+This method will populate results on the Sauce Labs Accessibility tab:
```
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -11,44 +11,91 @@ keywords:
+__Requirements__
+
+* [Install and Set up](https://webdriver.io/docs/gettingstarted/) WebdriverIO
+  * This documentation also shows you how to run a test locally which will help troubleshoot potential issues with your code before running on Sauce Labs
```
```suggestion
  * This documentation shows you how to run a test locally, and also how to troubleshoot potential issues with your code before running on Sauce Labs
```

titusfortner replied:
What about
> This documentation shows you how to run a test locally, which allows you to troubleshoot potential issues with your code before running on Sauce Labs

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
-The final step is to define a local variable to store the violations (critical, moderate, and/or minor) and return the `result`:
+__Usage__
 
+This method will populate results on the Sauce Labs Accessibility tab:
```
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
 
-<TabItem value="chrome">
+We highly recommend using the Python [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/).
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

spider-sauce commented on docs/basics/integrations/deque.md:
````diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
-    },
-];
-```
+The following is information on how to use the Python sa11y library directly:
````
```suggestion
The following is information on how to use the Python `sa11y` library directly:
```

spider-sauce commented on docs/basics/integrations/deque.md:
````diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
 
-```js
-browser.getAxeResults()
+This method will populate results on the Sauce Labs Accessibility tab:
````
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
-config.key = "yourAccessKey";
-config.region = "yourRegion -> us / eu";
+We highly recommend using the Ruby [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/). 
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
-config.capabilities = [
-    
-];
+The following is information on how to use the Ruby sa11y library directly:
```
```suggestion
The following is information on how to use the Ruby `sa11y` library directly:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
 
-* `us` - *default*
-* `eu`
+This method will populate results on the Sauce Labs Accessibility tab:
```
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

spider-sauce commented on docs/basics/integrations/deque.md:
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
-        platformName: 'macOS 10.15',
-        browserVersion: 'latest',
-    },
+This method will populate results on the Sauce Labs Accessibility tab:
```
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

spider-sauce reviewed (changes requested):
A couple of nitpicky passive voice and typos comments. Other than that it LGTM 👍
